definitions/divorce/json/CaseEvent.json

```diff
@@ -2703,6 +2703,17 @@
     "PostConditionState": "AmendPetition",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "29/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "amendPetitionForRefusalSol",
+    "Name": "Submit amended application",
+    "Description": "Submit amended application",
+    "DisplayOrder": 3,
```

Comment:
are you sure this is fine? I mean don't we have more than one "3" in that list now?

Reply:
@yarpo. Would reconfirm this is ok else update to more appropriate number. Thanks

Reply:
> are you sure this is fine? I mean don't we have more than one "3" in that list now?

@Dizrupto Should i be "worried" about the display index? AC's is quite about it but would be nice to use a reasonable value, thats if current in not reasonable.

Reply:
@yarpo I can confirm that for the list of Petitioner solicitor events there is no other with DisplayOrder: 3. I believe this index is fine in this case.

Reply:
Also, I think even if some have the same index CCD can handle that.